### DISCUSSION TITLE
Why is the E in MusE capitalized?

### DISCUSSION THREAD

**Discussion by Wallby** [muse-sequencer/muse#1270] [General]
Curious about the origin of the name. Wikipedia has no information about it currently.
https://en.wikipedia.org/wiki/MusE

**Comment by spamatica** [comment]
Good question!
I actually don't remember and the original author has long since moved on.

Thinking about it I think stands for Editor, e.g. Music Editor... no guarantee it is right though ;)

**Comment by minux22** [comment]
I first came across MusE around 2007. I can’t clearly remember whether it was on the official website or in the manual, but I recall seeing it described as ‘Linux [Mus]ic [E]ditor.’ 
In fact, on the https://sourceforge.net/projects/lmuse page, it is written as Linux Music Editor, and some people even refer to MusE as ‘lmuse.’
